ledger/acctupdates.go

```diff
@@ -879,6 +879,11 @@ func (aul *accountUpdatesLedgerEvaluator) GenesisHash() crypto.Digest {
 	return aul.au.ledger.GenesisHash()
 }
 
+// GenesisProto returns the genesis consensus params
+func (aul *accountUpdatesLedgerEvaluator) GenesisProto() config.ConsensusParams {
+	return aul.au.ledger.GenesisProto()
```

Comment:
where is it used?

Reply:
I added this while trying to merge recent ledger changes. I has aded GenesisProto to the new Ledger interface.  But after discussion with @tsachiherman , that interface may be gone now.  I'll look around some more and clean up.